python_modules/libraries/dagster-dg/dagster_dg/utils.py

```diff
@@ -288,24 +289,32 @@ def _format_error_message(message: str) -> str:
     return textwrap.fill(message, width=10000)
 
 
-NOT_DEPLOYMENT_ERROR_MESSAGE = _format_error_message("""
+def generate_missing_component_type_error_message(component_type_name: str) -> str:
+    return f"""
+        No component type named `{component_type_name}` is registered. Use 'dg component-type list'
+        to see the registered component types in your environment. You may need to install a package
+        that provides `{component_type_name}` into your environment.
+    """
```

Comment:
I think this is worth waiting on @OwenKephart's change to our naming scheme to include `@`. I think our verbiage will change here, will it not?

Reply:
I think "name" would be the component _before_ the `@` and some other word would describe the filename/package name. Curious where @OwenKephart's head is at on that.